src/services/emails/confirm-reset-password.js

```diff
@@ -1,29 +1,20 @@
-const Promise = require('bluebird')
-const config = require('../../config/config')
-const transporter = require('../../config/nodemailer')
-
-module.exports = (user, host) => {
-  let template = {
-    from: config.EMAIL,
-    to: user.email,
-    subject: 'Style Point: Reset your password',
-    html: `
+module.exports = params => {
+  let getTemplate = () => {
+    let template = {
+      subject: 'Style Point: Reset your password',
+      html: `
         <p>
             You are receiving this because you (or someone else) have requested the reset of the password for your account.
             <br>Please click on the following link, or paste this into your browser to complete the process:<br><br>
-            http://${host}/api/reset-password/${user.resetPasswordToken}<br><br>
+            http://${params[0]}/api/reset-password/${params[1]}<br><br>
```

Comment:
For better code readability, I'd rather see the properties used inside each string template. Using `params[]` makes you go through all the code just to find out what you're sending as parameters.